andela/octopus-bn-backend
# #170947446 Reject request (By Requester’s Manager)

### What does this PR do?

- Add and authenticate ```PUT/api/v1/trips/:tripId/reject``` route
- Allow managers to reject trips requests that are in pending mode
- Add documentation for a feature
- Add some validations
- Add French and English translation access


### How should this be manually tested?

- Run ```npm install ``` to setup necessary dependencies
- Run ``` npm run test ``` to see if everything is working as expected
- Run ``` npm run dev ``` to start local server
- Head over to postman and send a PUT request on this route: ```localhost:3000/api/v1/trips/:tripId/reject```

### Implementation plan

[Rejecting a request](https://docs.google.com/document/d/13XgZReJn4jw8v24GKu3nMysHDfyJP0SJ2Vx3_qDseg4/edit?usp=sharing)

### Stories associated with PR

[#170947446](https://www.pivotaltracker.com/story/show/#170947446)

### Screenshots if any
<img width="1125" alt="Screen Shot 2020-02-26 at 18 46 26" src="https://user-images.githubusercontent.com/51243858/75367050-5b7f6300-58c8-11ea-8c6b-a2505db341a1.png">
